app/src/components/modal/modal_deposit_withdraw/index.tsx

```diff
@@ -344,17 +339,19 @@ export const ModalDepositWithdraw = (props: Props) => {
           {exchangeType === ExchangeType.withdraw && currencySelected !== 'dai' && xDaiBalance?.isZero() ? (
             <InputInfo>
               <IconAlertInverted />
-              <div style={{ marginLeft: '12px' }}>Fund your Omen Account with Dai to proceed with the withdrawal.</div>
+              <BalanceItemBalance marginLeft={'12px'}>
+                Fund your Omen Account with Dai to proceed with the withdrawal.
+              </BalanceItemBalance>
```

Comment:
Not sure but this text doesn't seem to be the same color as in figma...And also we need to pay attention when reusing classes that it makes sense... For example here it would maybe make sense to name it something more general if we are going to use it for multiple stuff...

Reply:
In this case that naming didn't make sense yes.. will revert that one to a its TYPE here it's ok.

Reply:
the color on figma was wrong. It hsould have been text2 like in the code. All good